app/assets/stylesheets/finder_frontend.scss

```diff
@@ -308,4 +306,82 @@
       }
     }
   }
+
+  .facet-tags {
+    margin-top: $gutter-one-third;
+    margin-bottom: $gutter;
+  }
+
+  .facet-tags__group {
+    padding: 5px;
+
+    &:nth-child(odd) {
+      background-color: $grey-4;
+    }
+  }
+
+  .facet-tags__wrapper {
+    vertical-align: middle;
+    margin: 5px 0;
+
+    @include media(tablet) {
+      display: inline-block;
+    }
+  }
+
+  .facet-tags__wrapper:first-of-type .facet-tags__preposition:first-of-type {
```

Comment:
Does this rule need to apply to `.facet-tags__wrapper:first-of-type`? It seems like the bold font is only needed on `facet-tags__preposition`?

Reply:
Doing just `facet-tags__preposition:first-of-type` applies the styling to every `facet-tags__preposition` that is the first child of it's parent element, whereas we only want to add the bold styling to the first `facet-tags__preposition` within the first `facet-tags__wrapper`.

Otherwise you end up with this:
<img width="528" alt="screen shot 2018-12-20 at 11 48 22" src="https://user-images.githubusercontent.com/29889908/50283239-2ae2a480-044d-11e9-847a-e73bace8a1b7.png">